### PR TITLE
feat: define the develop and main branch to run the actions on commit…

### PR DESCRIPTION
**Pull request by clauBv23**
## Description

Changes addressed on this PR:

- Run actions over all PR relevant PR changes.
- Run the actions over the commit changes made on main.

Task ID: [OS-1134](https://aragonassociation.atlassian.net/browse/OS-1134)

## Checklist:

- [x] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

[OS-1134]: https://aragonassociation.atlassian.net/browse/OS-1134?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by banasa44 — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]

File: .github/workflows/sdk-tests.yml [hunk 1]
```diff
@@ -1,9 +1,17 @@
 name: SDK Tests
 
 on:
+  push:
+    branches:
+      - main
+      - develop
+
```
Comment (Rekard0) [inline .github/workflows/sdk-tests.yml:8]:
```suggestion
```

**Review by josemarinas — COMMENTED** [review]
Correct me if I'm wrong but thiw wont trigger on prs

**Comment by josemarinas** [comment]
> Correct me if I'm wrong but this wont trigger on prs

nevermind, I think we are ok

**Review by josemarinas — DISMISSED** [review]

**Comment by josemarinas** [comment]
just fyi @Rekard0 , this reverts the changes on https://github.com/aragon/osx-commons/pull/63/files

We deleted the restrictions so we tested that changes on one packages does not break the other ones, this addresses this by testing everything on `develop` and `main`, if there are breaking changes it will fail but it will do it **after** we merge the PR

**Comment by clauBv23** [comment]
> just fyi @Rekard0 , this reverts the changes on https://github.com/aragon/osx-commons/pull/63/files
> 


The change on that PR was done due to actions being triggered on changes on commits, not PRs (on `push` instead of on `pull_request`).
That was causing action not to run on certain commits and PR seemed correct when it wasn't, for example [here](https://github.com/aragon/osx/pull/527#discussion_r1491067746).

This PR is not reverting the changes made on PR #63. In the current approach, the modifications made on the defined path are scoped to the entire PR, not a single commit as was being done. 

Regarding the second

**Review by josemarinas — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]

**Pull request merged**